Realm/Tests/MigrationTests.mm

```diff
@@ -815,11 +815,21 @@ - (void)testEnumeratedObjectsDuringMigration {
 - (void)testMigrationBlockNotCalledForIntialRealmCreation {
     [RLMRealm setSchemaVersion:1
                 forRealmAtPath:RLMTestRealmPath()
-            withMigrationBlock:^(__unused RLMMigration *migration, __unused NSUInteger oldSchemaVersion) {
+            withMigrationBlock:^(__unused RLMMigration *migration, __unused uint64_t oldSchemaVersion) {
                 XCTFail(@"Migration block should not have been called");
             }];
     XCTAssertNoThrow([self realmWithTestPath]);
 }
 
-@end
+- (void)testRLMNotVersionedHasCorrectValue
```

Comment:
That `RLMNotVersioned` has a fixed, consistent value on all platforms that run the test.